daprdocs/content/en/developing-applications/building-blocks/pubsub/pubsub-overview.md

```diff
@@ -3,49 +3,64 @@ type: docs
 
-The Dapr pub/sub building block provides a platform-agnostic API to send and receive messages. Your services publish messages to a named topic and also subscribe to a topic to consume the messages.
+## Pub/sub API in Dapr
 
-The service makes a network call to a Dapr pub/sub building block, exposed as a sidecar. This building block then makes calls into a Dapr pub/sub component that encapsulates a specific message broker product. To receive topics, Dapr subscribes to the Dapr pub/sub component on behalf of your service and delivers the messages to an endpoint when they arrive.
+The Pub/sub API in Dapr:
+- Provides a platform-agnostic API to send and receive messages.
+- Offers an at-least-once guarantee.
+- Integrates with various message brokers and queuing systems. 
 
-The diagram below shows an example of a "shipping" service and an "email" service that have both subscribed to topics that are published by the "cart" service. Each service loads pub/sub component configuration files that point to the same pub/sub message bus component, for example Redis Streams, NATS Streaming, Azure Service Bus, or GCP Pub/Sub.
+The specific implementation used by your service is pluggable and configured as a Dapr Pub/sub component at runtime. This approach removes the dependency from your service and, as a result, makes your service more portable and flexible to changes.
+
+When using the Pub/sub API in Dapr:
+
+1. Your service makes a network call to a Dapr Pub/sub building block, exposed as a sidecar.
+1. The Pub/sub building block makes calls into a Dapr Pub/sub component that encapsulates a specific message broker product.
+1. To receive topics, Dapr subscribes to the Dapr Pub/sub component on behalf of your service and delivers the messages to an endpoint when they arrive.
+
+In the diagram below, a "shipping" service and an "email" service have both subscribed to topics published by a "cart" service. Each service loads Pub/sub component configuration files that point to the same Pub/sub message bus component; for example: Redis Streams, NATS Streaming, Azure Service Bus, or GCP Pub/Sub.
 
 <img src="/images/pubsub-overview-components.png" width=1000>
 <br></br>
 
-The diagram below has the same services, however this time showing the Dapr publish API that sends an "order" topic and order endpoints on the subscribing services that these topic messages are delivered posted to by Dapr.
+In the diagram below, the Dapr API posts an "order" topic from the publishing "cart" service to "order" endpoints on the "shipping" and "email" subscribing services.
 
 <img src="/images/pubsub-overview-publish-API.png" width=1000>
 <br></br>
 
-## Features
-The pub/sub building block provides several features to your application.
+[View the complete list of Pub/sub components that Dapr supports]({{< ref supported-pubsub >}}).
+
+## Dapr Pub/sub API features
+
+The Pub/sub building block brings several features to your application.
 
 ### Cloud Events message format
```

Comment:
At the end of this whole section on Cloud Events, I would resolve the issue here, but then move most of the added details into a separate article just called Cloud events. Effectively see the custom cloud event article here https://github.com/dapr/docs/pull/2414/files, just do not call it Custom, this is simply about Understand Dapr and CloudEvents, and maybe that is also the title.

Reply:
Made a note of this and will include in a different PR!